src/index.tsx

```diff
@@ -16,7 +16,7 @@ export interface Props {
 
 const VisibilitySensor: React.FC<Props> = (props) => {
   const myView: any = useRef(null);
-  const [lastValue, setLastValue] = useState<boolean>(false);
+  const [lastValue, setLastValue] = useState<boolean>(null);
```

Comment:
👋🏽  Argument of type 'null' is not assignable to parameter of type 'boolean'. Why can't we stick with `false`?

Reply:
just following from this https://github.com/SvanBoxel/visibility-sensor-react-native/issues/4#issue-925818433